Userland/DevTools/UserspaceEmulator/Emulator.cpp

```diff
@@ -532,6 +532,14 @@ bool Emulator::find_malloc_symbols(const MmapRegion& libc_text)
     m_calloc_symbol_end = m_calloc_symbol_start + calloc_symbol.value().size();
     m_malloc_size_symbol_start = malloc_size_symbol.value().value() + libc_text.base();
     m_malloc_size_symbol_end = m_malloc_size_symbol_start + malloc_size_symbol.value().size();
+
+    reportln("=={}==  Mapped Malloc and Free", getpid());
+    reportln("    malloc: {:p} - {:p}", m_malloc_symbol_start, m_malloc_symbol_end);
+    reportln("    free: {:p} - {:p}", m_free_symbol_start, m_free_symbol_end);
+    reportln("    realloc: {:p} - {:p}", m_realloc_symbol_start, m_realloc_symbol_end);
+    reportln("    calloc: {:p} - {:p}", m_calloc_symbol_start, m_calloc_symbol_end);
+    reportln("    malloc_size: {:p} - {:p}", m_malloc_size_symbol_start, m_malloc_size_symbol_end);
```

Comment:
This seems like `dbgln()` material, not `reportln()` material.

Reply:
Should I flag it with something too?

Reply:
Do we even need it at this point, after the error this was realted to was resoved?

Reply:
Oh good point. Yeah then we can just drop it :)

Reply:
Ok done that then